src/Routes.js

```diff
@@ -18,3 +21,7 @@ export default function Routes() {
     </Switch>
   )
 }
+
+export function getPreferencesSearch(screen) {
```

Comment:
We didn't use react-router in the client, but I wonder if there's a "framework appropriate" way to handle this with react-router.